paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerAdapter.kt

```diff
@@ -169,7 +171,7 @@ fun interface CustomerEphemeralKeyProvider {
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 fun interface SetupIntentClientSecretProvider {
 
-    suspend fun provide(customerId: String): Result<String>
+    suspend fun provide(customerId: String): CustomerAdapterResult<String>
```

Comment:
Nit: Can we rename this `provideSetupIntentClientSecret`? Makes things clearer if a developer implements this method with an existing class, e.g. their ViewModel.